jenkins_pipelines/manager_prs/Jenkinsfile_prs_ruby_rubocop

```diff
@@ -49,9 +52,13 @@ pipeline {
                 echo 'Run tests!'
                 script {
                     commands = "${gitarro_cmd} ${runtest}"
```

Comment:
Not for this PR (in fact I should have commented this before), but why not having:
```
                    if (params.GITARRO_PR_NUMBER != '') {
                            commands = "${gitarro_local} ${runtest}"
                    }
                    else
                            commands = "${gitarro_cmd} ${runtest}"
                    }
```
?

Reply:
Just because if we have that, we need to have a previous line declaring the variable with something. Otherwise, the pipeline will not be happy.